### PR TITLE
Update Kubernetes clusters to v1.4.6

### PR DESCRIPTION
**Pull request by dghubble**
* Upgrade Kubernetes v1.4.6 
    * static manifest clusters (k8s, k8s-install)
    * static manifest, using rkt runtime (rktnetes, rktnetes-install)
    * self-hosted Kubernetes (bootkube, bootkube-install)

cc @pbx0 @aaronlevy

### CODE REVIEW

File: Documentation/rktnetes.md [hunk 1]
```diff
@@ -46,16 +46,16 @@ Client machines should boot and provision themselves. Local client VMs should ne
 
 [Install kubectl](https://coreos.com/kubernetes/docs/latest/configure-kubectl.html) on your laptop. Use the generated kubeconfig to access the Kubernetes cluster created on rkt `metal0` or `docker0`.
 
-    $ cd /path/to/coreos-baremetal
-    $ kubectl --kubeconfig=examples/assets/tls/kubeconfig get nodes
+    $ KUBECONFIG=examples/assets/tls/kubeconfig
```
Comment (aaronlevy) [inline Documentation/rktnetes.md]: export KUBECONFIG?

Comment (dghubble) [inline Documentation/rktnetes.md]: Shouldn't be needed, no scripts or subshells in this project read from it

Comment (aaronlevy) [inline Documentation/rktnetes.md]: But none of the commands below will respect this env var...

Comment (aaronlevy) [inline Documentation/rktnetes.md]: nm ignore me

**Pull request merged**